<?xml version="1.0"?>
<launch>

    <node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen" respawn="true">
    <!-- node parameters -->
    <param name="serial_name" type="string" value="/dev/ttyS1"/>
    <param name="baud_rate" type="int" value="230400"/>
    <param name="app_id" type="int" value="1073792"/>
    <param name="app_version" type="int" value="1"/>
    <param name="align_time" type="bool" value="false"/>
    <param name="enc_key" type="string" value="49f507cdc0209ecf18cb6b1ee32408bff3e58e82214f664e893a3de70a523662"/>
    <param name="use_broadcast" type="bool" value="false"/>
    </node>

  <!-- Gets attitude information and converts to transforms -->
 <node pkg="gpar_dji" type="telemetry" name="telemetry" output="screen" />

  <!-- Tracks information from MSDK and process them -->
 <node pkg="gpar_dji" type="mobile_comm" name="mobile_comm_parser" /> 

  <!-- roda mission waypoint -->
  <node pkg="gpar_dji" type="waypoints" name="waypoints" output="screen" respawn="true"/>

  <!-- Roda Supervisor -->
  <node pkg="gpar_dji" type="waypoints_supervisor" name="waypoints_supervisor" output="screen" required="true"/>

  <!-- Roda Cloud Controller -->
  <node pkg="gpar_lidar" type="cloud_controller" name="cloud_controller" output="screen" >
	  <param name="input_cloud" type="str" value="cloud_ldmrs" />
  </node>

 <!-- Roda BagFile
 <node pkg="rosbag" type="record" name="rosbag_record_scan"
	 args="-o /home/linaro/drone_scan /cloud_ldmrs /tf /tf_static" />
	-->



</launch>

